packages/migrate/namedModulesPlugin/__tests__/namedModulesPlugin.test.ts

```diff
@@ -0,0 +1,5 @@
+import defineTest from "@webpack-cli/utils/defineTest";
+import { join } from "path";
+defineTest(join(__dirname, ".."), "namedModulesPlugin", "namedModulesPlugin-0");
```

Comment:
path into variable

Reply:
Is it trivial?

Reply:
doesn't look readable for people maintaining this